File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        size_t length = nums.size();

        for (int i = 0; i < length; i++) {
            int num1 = nums[i];
            for (int j = i+1; j < length; j++) {
                if (num1 == nums[j]) return true;
            }
        }
        return false;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        res = defaultdict(int)

        for i in nums:
            res[i] += 1

        ans = []
        for key, value in res.items():
            ans.append((value, key))

        ans.sort(reverse = True)
        return_val = []
        for i in range(k):
            return_val.append(ans[i][1])
        return return_val